tests/unit/zhmcclient/test_partition.py

```diff
@@ -18,14 +18,14 @@
 
 from __future__ import absolute_import, print_function
 
-import re
 import copy
+import re
+
 import pytest
+from tests.common.utils import assert_resources
```

Comment:
The order of imports so far was grouped into these groups, in order:
* built-in modules
* installed modules
* zhmcclient
* tests

There is no particular technical reason for that; it is just ordered from global to local.

`tests` is a bit special because it is just a local directory but occupies that name in the namespace of packages.

I prefer if we can keep this grouping.

Reply:
I just did a formatting of file on my pycharm IDE, reverted it back and will follow the above instructions from now on